### DISCUSSION TITLE
stl for openfoam meshing and boundary set

### DISCUSSION THREAD

**Discussion by WHUT666** [nextfoam/baram#28] [Ideas]
how to import stl file and set boundary conditions，then use blockmesh and snappyHexmesh to generate the mesh？

**Comment by jiban** [comment]
BARAM Meshing Tool is scheduled to be developed next year.
Currently BARAM supports importing following meshes only.
* OpenFOAM polyMesh ( resides under _constant_ folder )
* Fluent Mesh `*.msh`
* Fluent Case `*.cas`
* Gmsh `*.msh`
* I-deas Universal `*.unv`

![image](https://user-images.githubusercontent.com/1674177/209116989-18f57a22-a926-44fa-95c9-bd5f45c09c1c.png)

**Reply by WHUT666** [reply]
Thank you very much，I will try to develop it.
